### DISCUSSION TITLE
Compress cookies

### DISCUSSION THREAD

**Discussion by hihellobolke** [OpenIDC/mod_auth_openidc#1010] [Ideas]
With claims client cookies can become quite large. I believe compressing cookies would help, so we can keep total headersize low.
I see OAuth2Proxy also compress their cookies.

https://github.com/oauth2-proxy/oauth2-proxy/blob/13202fd5ea94be3f3d29cb4aca8af7d8c3ae666a/pkg/apis/sessions/session_state.go#L172

For mod auth openidc it maybe good to compress after JWE encryption of session cookies.

**Comment by zandbelt** [comment]
zlib (or compile time configurable brotli) based compression is available in version >= 2.4.13
